docs/application-tutorial/game_of_life.py

```diff
@@ -0,0 +1,511 @@
+import sys
+import time
+import math
+import numpy as np
+
+# Hide pygame prompt
+from os import environ
+
+environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
+
+import pygame
+import argparse
+
+from concrete import fhe
+
+# Constants of the game of life
+weights_method_3b_a = np.array([[1, 1, 1], [1, 0, 1], [1, 1, 0]])
+weights_method_3b_b = np.array([[0, 0, 0], [0, 0, 0], [0, 0, 1]])
+table_next_cell_3b_a = [i if i <= 3 else 4 for i in range(8)]
+table_next_cell_3b_b = [i - 1 if i in [2, 3] else 0 for i in range(6)]
+table_next_cell_3b_c = [i in [2, 3] for i in range(4)]
```

Comment:
What does abc refer to here? Could you explain a bit with comments maybe?

Reply:
That's the first, second and third table, just that. You would prefer _one, _two and _three, or some other convention, or comments?

Reply:
Please tell me if I can close, or what you prefer

Reply:
no meaning, it's just that I have several tables in the algo. I don't think it would bring that much to give more precise names. 

eg, things like:

```
    grid_b = grid_a + convoluted_grid

    # Apply a TLU: input in [0, 5], output in [0, 4]
    grid_c = table_cp_next_cell_3b_b[grid_b]

    # Add center
    grid = grid_c + grid

    # And a last TLU: input in [0, 5] and output in [0, 1]
    grid = table_cp_next_cell_3b_c[grid]
```

Reply:
If the naming can't be explicit, it's fine like this :+1: